packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -192,6 +197,11 @@ const AppModals = observer(() => {
     } else if (isUrlUnavailableModalVisible) {
         ComponentToLoad = <UrlUnavailableModal />;
     }
+
+    if (is_logged_in && show_effortless_login_modal && is_mobile) {
```

Comment:
cant we move these conditions to `useShowEffortlessLoginModal` hook itself instead of checking multiple conditions here?

Reply:
@amina-deriv, later we will add passkeys also for desktop, and will remove 'is_mobile'.
and I believe this modal is temporary implementation. I thinks there is no need to add additional hooks

Reply:
Not a new hook @yauheni-deriv 
Cant we move these `is_logged_in and is_mobile`  to `useShowEffortlessLoginModal`  and return true from hook based on all these. In this way  all conditions will be handled in single place.

Reply:
oh, I got you, @amina-deriv.
I'm not sure, we are using here `is_logged_in and is_mobile` coming from store, and the hook is in 'hooks' package...

Reply:
In hooks package, we can use storevalues

Reply:
sure @amina-deriv, will check it 👍